packages-native/actions/src/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
What is the reason for having 3 tsconfigs related to actions?

Reply:
Because one is to deliver, and the other is for the te files for copying

Reply:
I understand, there is another one though in the actions folder than can be removed.